.travis.yml

```diff
@@ -1,25 +1,24 @@
 addons:
-  firefox: "49.0.2"
+  firefox: "54.0.1"
 
+dist: trusty
 language: ruby
-sudo: false
 
 cache:
   directories:
     - "travis_geckodriver"
 
 rvm:
-  - '2.3.1'
+  - '2.4.1'
 
 before_script:
   - if [ ! -d $PWD/travis_geckodriver ];  then mkdir $PWD/travis_geckodriver; fi
-  - if [ ! -f $PWD/travis_geckodriver/geckodriver ]; then wget https://github.com/mozilla/geckodriver/releases/download/v0.11.1/geckodriver-v0.11.1-linux64.tar.gz; fi
-  - if [ ! -f $PWD/travis_geckodriver/geckodriver ]; then tar -xvf geckodriver-v0.11.1-linux64.tar.gz -C $PWD/travis_geckodriver; fi
+  - if [ ! -f $PWD/travis_geckodriver/geckodriver ]; then wget https://github.com/mozilla/geckodriver/releases/download/v0.16.0/geckodriver-v0.16.0-linux64.tar.gz; fi
```

Comment:
I found a note in a selenium changelog somewhere that "geckodriver 0.16.0 is strongly recommended"; newer versions reliably break in CI